<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>svrcClass</class>
 <widget class="QMainWindow" name="svrcClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>svrc</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="roomLabel">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>51</y>
      <width>39</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>房间号</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="demoGroupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>子示例</string>
    </property>
    <widget class="QListWidget" name="demoListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="schedulePushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>51</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>安排房间</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="descLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>261</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请输入已存在房间号，选择某个示例，然后点击启动按钮。
如果房间不存在，请先点击+按钮安排房间。</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="roomLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>52</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>r16</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>roomLineEdit</tabstop>
  <tabstop>demoListWidget</tabstop>
  <tabstop>startBtn</tabstop>
  <tabstop>schedulePushButton</tabstop>
 </tabstops>
 <resources>
  <include location="svrc.qrc"/>
 </resources>
 <connections/>
</ui>
